# Array of tracked persons with their information

std_msgs/Header header

# Array of tracked persons
PersonInfo[] persons

# Total number of currently tracked persons
int32 num_persons
